sabelotodo/tests/fixtures.py

```diff
@@ -59,15 +59,24 @@ def app(database):
     return app
 
 
-@pytest.fixture(scope='session')
-def _db(app):
+@pytest.fixture(scope='function')
+def _db(app, request):
     '''
     Provide the transactional fixtures with access to the database via a Flask-SQLAlchemy
     database connection.
```

Comment:
Let's add a comment to this docstring explaining that the tables are dropped after each test.